frontend/lib/norent/letter-content.tsx

```diff
@@ -85,17 +87,26 @@ const LetterHeading: React.FC<NorentLetterContentProps> = (props) => (
   </dl>
 );
 
-const TenantProtections: React.FC<{}> = () => (
-  <>
-    <p>
-      Tenants impacted by the COVID-19 crisis are protected from eviction for
-      nonpayment per emergency declaration(s) from:
-    </p>
-    <ul>
-      <li>US Congress, CARES Act (Title IV, Sec. 4024), March 27, 2020</li>
-    </ul>
-  </>
-);
+const TenantProtections: React.FC<NorentLetterContentProps> = (props) => {
+  const state = props.state as USStateChoice;
+  const protectionData = getNorentMetadataForUSState(state)?.lawForLetter;
+  const protectionEntries = Object.entries(protectionData);
```

Comment:
Hmm, something to keep in mind with this strategy is that it might not always be the case that the entries are in the order that we wanted them to be in, i.e. from 1 to 6.  Also, if that airtable ever has additional fields added to it, they might inadvertently end up getting listed here... I think it might be best for me to modify the `national-metadata.tsx` at some point to just explicitly turn the texts 1-6 into a proper array, and for this method to just iterate through the array.  I guess it's probably not a big deal in the end tho.